.circleci/main.yml

```diff
@@ -1414,15 +1417,25 @@ jobs:
       - run: #Note: --no-ansi https://github.com/python-poetry/poetry/issues/7184
           name: Install Python Dependencies with Poetry
           command: poetry --no-ansi install
-      - install-python-test-deps
-
-      # export PACH_PYTHON_ENTERPRISE_CODE=${ENT_ACT_CODE}
-      # export export PACH_PYTHON_AUTH_TOKEN=iamroot
       - run:
-          name: Run Tests
+          name: Start minikube
+          command: minikube start --vm-driver=docker --kubernetes-version=v1.19.0 --cpus=7 --memory=12g --wait=all
```

Comment:
For this, we could use the main start minikube circle command and then add a task to the following shell script to wait for minikube
https://github.com/pachyderm/pachyderm/blob/master/etc/testing/circle/wait-minikube.sh